src/core/raster/qgscolorrampshader.h

```diff
@@ -72,6 +72,12 @@ class CORE_EXPORT QgsColorRampShader : public QgsRasterShaderFunction
     /** \brief Get the color ramp type */
     QgsColorRampShader::ColorRamp_TYPE colorRampType() const {return mColorRampType;}
 
+    /** \brief Get the original color ramp name
+     * @note added in QGIS 3.0
+     * @see setColorRampName()
+     */
+    QString colorRampName() const {return mColorRampName;}
```

Comment:
Seems this case does not handle by `prepare-commit.sh` and `astyle.sh` scripts. I noticed same formatting in other filed. Changed it manually.